src/styles/stars.scss

```diff
@@ -7,7 +7,12 @@
     margin-right: 4px;
     background: url(../images/star-active.svg) center no-repeat;
 
-    &:last-of-type {
+    .stars--0 &:nth-of-type(n+1),
+    .stars--1 &:nth-of-type(n+2),
+    .stars--2 &:nth-of-type(n+3),
+    .stars--3 &:nth-of-type(n+4),
+    .stars--4 &:nth-of-type(n+5),
+    .stars--5 &:nth-of-type(n+6) {
```

Comment:
Will be nice to use [cycle](https://sass-lang.com/documentation/at-rules/control/for)